tcms/testruns/models.py

```diff
@@ -208,7 +208,7 @@ def remove_cc(self, user):
     def remove_env_value(self, env_value):
         EnvRunValueMap.objects.filter(run=self, value=env_value).delete()
 
-    def mail(self, template, subject, context, to=[], request=None):
+    def mail(self, template, subject, context, request=None):
```

Comment:
It looks like this method is only used in `tcms.signals.handle_emails_post_run_save` so you can remove it entirely and place the sending code there.